Update ERC-7829: Move to Review

Allen2730 opened the pull request.

SamWilsn reviewed (commented):
Woops, submitted early. Here's the rest of my review:

- What's a "valiad"?
- Make sure any code snippet (eg. "transfer, approve, setReader" on L101) are in backticks (\`).
- On L125, use "ERC-721" and not just "721".
- L119 isn't really proper Rationale content. This paragraph is explaining why you need ERC-7829, and so belongs in Motivation. The Rationale section is for explaining technical choices made within the proposal.

xinbenlv reviewed (changes requested):
Please research into existing ERCs for relevant topics and add to the motivation

Allen2730 commented:
> Woops, submitted early. Here's the rest of my review:
> 
> * What's a "valiad"?
> * Make sure any code snippet (eg. "transfer, approve, setReader" on L101) are in backticks (`).
> * On L125, use "ERC-721" and not just "721".
> * L119 isn't really proper Rationale content. This paragraph is explaining why you need ERC-7829, and so belongs in Motivation. The Rationale section is for explaining technical choices made within the proposal.

Based on your suggestions, we have made the following changes:

- Changed valiad to valid.
- Used backticks in all code snippets.
- Changed to ERC-721.
- Updated the Rationale section.

Thanks for your suggestion.

SamWilsn commented on ERCS/erc-7829.md:
```diff
@@ -13,7 +13,9 @@ requires: 165, 721
 ## Abstract
 
-This proposal is an extension of [ERC-721](./eip-721.md). This proposal introduces a new role, Reader, who is granted by the Owner and allows multiple Readers for a single NFT. Additionally, this proposal extends the metadata interface of ERC-721, requiring that metadata should include at least a commitment, size, expire time, and uploader's address. This proposal also proposes a storage proof me ...
+This proposal extends the [ERC-721](./eip-721.md) standard to support Data Assets which refers to digital products created by creators.
```
> [...] Data Assets which refers to digital products created by creators.

A bit of a nitpick, but anything created is created by a creator ;)
